scipy/_lib/meson.build

```diff
@@ -287,6 +287,7 @@ py3.install_sources(
 py3.install_sources(
   [
     'pyprima/prima/pyprima/src/pyprima/__init__.py',
+    'pyprima/prima/LICENCE.txt',
```

Comment:
This file should not be installed. No other license files are. They will instead end up in the `dist-info` directory in the future (once PEP 639 support is out). There's no problem today; there is no actual requirement to install license files, only to ship them.

Reply:
Ok, can remove. How are we shipping? Do we already meet that ba, or do I need to make a change?

Reply:
Should the licenses_bundled file point to where they are in the source, or installed?

Reply:
> How are we shipping? Do we already meet that bar, or do I need to make a change?

sdists automatically include license files. wheels rely on `LICENSES_bundled.txt` being appended to `LICENSE.txt`. No change needed.

> Should the licenses_bundled file point to where they are in the source, or installed?

The source location.